storage/storage_image.go

```diff
@@ -1207,7 +1207,7 @@ func (s *storageImageDestination) Commit(ctx context.Context, unparsedToplevel t
 		if len(oldNames) > 0 {
```

Comment:
The `oldNames` variable is now conceptually unnecessary and should be eliminated (_maybe_ apart from the error message about cleanup).

Reply:
Actually i wanted to be change to be minimum like just `s/SetNames/AddNames` but you are right conceptually we don't need `oldNames` anymore.

Reply:
FWIW the way I think about this, there will never be more time or more motivation to clean up and simplify the code.

Keeping the code as simple and as easy to maintain as possible is an everyday job.